src/methods.jl

```diff
@@ -38,3 +38,44 @@ julia> bias_subtraction(frame, bias)
 * [`bias_subtraction!`](@ref)
 """
 bias_subtraction(frame::AbstractArray, bias_frame::AbstractArray) = bias_subtraction!(deepcopy(frame), bias_frame)
+
+
+#=
+***Note to future developer***
+The code for overscan_subtraction is not type-stable because of splat operation.
```

Comment:
Does returning `frame` make the function type-stable? I don't see why it should be unstable

Reply:
Yes, it worked!
I replaced 
`return frame .-= overscan_value` with 
```
frame .-= overscan_value
return frame
```
But how did it work? :thinking:

Reply:
I'm not at the computer, so I couldn't test it, but my guess is that it happened because assignment returns the right-hand side value: https://giordano.github.io/blog/2017-12-02-julia-assignment/.

It's a bit weird to return an assignment, usually it's just useless to assign a value to a variable and never use it again later. In this case you were modifying in-place an object, but probably returning an unexpected thing